cranelift/filetests/filetests/isa/x86/windows_fastcall_x64.clif

```diff
@@ -94,3 +135,100 @@ block0(v0: i64):
     store.i64 v9, v0+72
     return v10
 }
+; check:  function %internal_stack_a(i64 [%rcx], i64 fp [%rbp], i64 csr [%r12], i64 csr [%r13], i64 csr [%r14], i64 csr [%r15]) -> i64 [%rax], i64 fp [%rbp], i64 csr [%r12], i64 csr [%r13], i64 csr [%r14], i64 csr [%r15] windows_fastcall {
```

Comment:
:tada: :tada:

Reply:
It turns out asserting things in tests is good :smile:.